frame/example/src/lib.rs

```diff
@@ -642,12 +643,50 @@ impl<T: Trait + Send + Sync> SignedExtension for WatchDummy<T> {
 	}
 }
 
+benchmarks!{
+	_ {
+		// Define a common range for `b`.
+		let b in 1 .. 1000 => ();
+	}
+
+	// This will measure the execution time of `accumulate_dummy` for b in [1..1000] range.
+	accumulate_dummy {
+		let b in ...;
+		let caller = account("caller", 0, 0);
+	}: _ (RawOrigin::Signed(caller), b.into())
+
+	// This will measure the execution time of `set_dummy` for b in [1..1000] range.
+	set_dummy {
+		let b in ...;
+		let caller = account("caller", 0, 0);
+	}: set_dummy (RawOrigin::Signed(caller), b.into())
+
+	// This will measure the execution time of `set_dummy` for b in [1..10] range.
+	another_set_dummy {
+		let b in 1 .. 10;
+		let caller = account("caller", 0, 0);
+	}: set_dummy (RawOrigin::Signed(caller), b.into())
+
+	// This will measure the execution time of sorting a vector.
+	sort_vector {
+		let x in 0 .. 1;
```

Comment:
```suggestion
		let x in 0 .. 10000;
```

This makes the benchmark actually do something :)

Reply:
I didn't want it to spend much time in the test, it's just an example. But I guess 10000 it's not much either.